titus_isolate/constants.py

```diff
@@ -0,0 +1,2 @@
+ALLOCATOR_CONFIG_CHANGE_EXIT = 1
```

Comment:
just curious: is the error code of the service being read & interpreted by the agent right now?

Reply:
if so, shouldn't this one be `0`?

Reply:
This is the error code that `systemd` will see.  I tried `0`.  If you use that, the service is viewed as having exited successfully and is *not* restarted, so a non-zero exit code is needed.